packages/bundle/package.json

```diff
@@ -0,0 +1,54 @@
+{
+  "name": "bundle",
+  "version": "1.0.0",
+  "description": "",
+  "main": "lib/index.js",
+  "private": true,
+  "scripts": {
+    "build": "concurrently --names \"babel,typecheck\" \"npm run build:babel\" \"npm run build:typecheck\"",
+    "build:babel": "babel --extensions .js,.ts,.tsx --ignore **/*.spec.js,**/*.spec.ts,**/*.spec.tsx,**/*.test.js,**/*.test.ts,**/*.test.tsx,__test__/**/*.js,__test__/**/*.ts,__test__/**/*.tsx --out-dir lib --source-maps inline --verbose src",
+    "build:typecheck": "tsc",
+    "clean": "npm run clean:dist && npm run clean:lib",
+    "clean:dist": "rimraf dist",
+    "clean:lib": "rimraf lib",
+    "prepublishOnly": "npm run clean && npm run build && npm run webpack",
+    "test": "jest",
+    "watch": "concurrently --names \"babel,typecheck\" \"npm run build:babel -- --watch\" \"npm run build:typecheck -- --preserveWatchOutput --watch\"",
+    "webpack": "webpack --run-prod",
+    "webpack-dev": "webpack --run-dev"
+  },
+  "dependencies": {
+    "@babel/runtime": "^7.0.0",
+    "babel-runtime": "^7.0.0-beta.3",
+    "backend": "^1.0.0",
+    "bluebird": "^3.5.2",
+    "botframework-directlinejs": "0.9.18-master.3cece81",
```

Comment:
correct version of directlinejs?

Reply:
You are really good at PR. 😊

As you know, DLJS and Web Chat are pretty-coupled together now. We are using tagged build before GA so we can move them as fast as we could.

Agree we should move it to a production build when we launch in preview. Will fix it.